index.html

```diff
@@ -4,86 +4,134 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta name="description" content="An aspiring software developer with experience in desktop, web & mobile applications.">
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide|Sofia|Trirong|Crete Round|Inter">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Audiowide|Sofia|Trirong|Crete Round|Inter|Roboto|Poppins">
         <title>Portfolio</title>
         <link rel="stylesheet" href="styles.css">
     </head>
     <body>
-        <div id="home-screen">
-            <nav id="navbar">
+        <header class="header">
+            <nav id="navbar" class="navbar">
                 <a href="">Md Tariq-ul Islam</a>
-                <a href=""><img src="./icons/Menu.svg"></a>
+                <a id="menu-icon" href=""><img src="./icons/Menu.svg"></a>
+                <ul id="menu-list">
+                    <li><a class="header-menu" href="">Portfolio</a></li>
+                    <li><a class="header-menu" href="">About</a></li>
+                    <li><a class="header-menu" href="">Contact</a></li>
+                    <li><a class="header-menu" href=""><img src="./icons/Mail.svg"></a></li>
+                </ul>                
             </nav>
-            <div id="welcome">          
```

Comment:
- [ ] Consider using an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
Hi @Kamba56,
Thanks for your comments.  👍👍👍